src/Keboola/ConfigMigrationTool/Service/OAuthService.php

```diff
@@ -12,10 +12,10 @@ class OAuthService
     /** @var Client */
     private $client;
 
-    public function __construct()
+    public function __construct(string $baseUrl = 'https://syrup.keboola.com/oauth-v2/')
```

Comment:
ten default bych dal doprdele, pouziva se to uz jen v jedny dalsi migraci, tak bud to tam pridej, anebo ji smaz jestli uz nefunguje